packages/index.yaml

```diff
@@ -46,6 +46,7 @@ packages:
   - iconUrl: https://avatars.githubusercontent.com/u/13629408
     latestVersion: v1.9.5+1
     name: ingress-nginx
+    scope: Namespaced
```

Comment:
@kosmoz  I am unsure if the namespace scope needs to be added here, since the `v1.9.5+1` doesn't have namespace scope anymore. Is this also automatically updated with the latestVersion?

Reply:
It would be updated by the bot but it's better to do it in the same PR because otherwise there is a discrepancy between the index and package.yaml